src/internal/scheduler/timerHandle.ts

```diff
@@ -0,0 +1 @@
+export type TimerHandle = number | NodeJS.Timeout;
```

Comment:
fancy. I think I usually just do `ReturnType<typeof setTimeout>`. LOL... because I don't ever trust that I'm right.